tests/Utils/Test_IPConverter.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+* Copyright (c) 2018 Zilliqa 
+* This source code is being disclosed to you solely for the purpose of your participation in 
+* testing Zilliqa. You may view, compile and run the code for that purpose and pursuant to 
+* the protocols and algorithms that are programmed into, and intended by, the code. You may 
+* not do anything else with the code without express permission from Zilliqa Research Pte. Ltd., 
+* including modifying or publishing the code (or any part of it), and developing or forming 
+* another public or private blockchain network. This source code is provided ‘as is’ and no 
+* warranties are given as to title or non-infringement, merchantability or fitness for purpose 
+* and, to the extent permitted by law, all liability for your use of the code is disclaimed. 
+* Some programs in this code are governed by the GNU General Public License v3.0 (available at 
+* https://www.gnu.org/licenses/gpl-3.0.en.html) (‘GPLv3’). The programs that are governed by 
+* GPLv3.0 are those programs that are located in the folders src/depends and tests/depends 
+* and which include a reference to GPLv3 in their program files.
+**/
+
+#include "libUtils/IPConverter.h"
+#include "libUtils/Logger.h"
+#include <string>
+
+#define BOOST_TEST_MODULE ipconverter
+#define BOOST_TEST_DYN_LINK
+#include <boost/multiprecision/cpp_int.hpp>
+#include <boost/test/included/unit_test.hpp>
+
+using namespace std;
+
+BOOST_AUTO_TEST_SUITE(ipconverter)
+
+BOOST_AUTO_TEST_CASE(test_IPNumercialtoString)
+{
+    INIT_STDOUT_LOGGER();
+    string ipStr = IPConverter::ToStrFromNumercialIP(
+        (boost::multiprecision::uint128_t)16777343);
+    BOOST_CHECK_EQUAL(ipStr, "127.0.0.1");
+}
+
+BOOST_AUTO_TEST_CASE(test_IPStringToNumercial)
+{
+    INIT_STDOUT_LOGGER();
+    boost::multiprecision::uint128_t ipNumercial
+        = IPConverter::ToNumercialIPFromStr("127.0.0.1");
+    BOOST_CHECK_EQUAL(ipNumercial, 16777343);
```

Comment:
Suggest to use BOOST_CHECK_MESSAGE(16777343 == IPConverter::ToNumercialIPFromStr("127.0.0.1"), "Please put some error message here"), in order to avoid using extra variable and give some error message if checking failed.